### PR TITLE
fixed issue with not working tos modal

### PR DESCRIPTION
**Pull request by cgibsonmm**
In order for the terms of service modal to work correctly I moved the render partial out of sidebar/combo and placed the render on the layout file, the tos now works correctly and is displaying

### CODE REVIEW

**Comment by jhsu802701** [comment]
Please sync this branch.  (Merge conflicts have been getting in the way.)

**Comment by jhsu802701** [comment]
The changes I've made since you submitted this pull request were simply upgrading Ruby to 2.6.3 and upgrading the nokogiri and ffi gems.

**Comment by jhsu802701** [comment]
OK, I think I know where the problem comes from.  I see that the Gemfile specifies for the development environment libnotify for Linux only and growl for Darwin (Mac OS) only.  I'm guessing that you're doing your Rails development in Mac OS, while my development environment and Travis CI use Linux.

**Comment by cgibsonmm** [comment]
I will take care of this in the am!

**Comment by jhsu802701** [comment]
I've made changes manually in my local development environment.  I'm able to get things to work by simply reversing the changes in the Gemfile.lock file.

**Comment by jhsu802701** [comment]
Corey, here are the changes I need:
1.  Sync the branch you're submitting
2.  Reverse the changes to Gemfile.lock at https://github.com/OpenTwinCities/adopt-a-tree/pull/228/commits/991a9638abc37d7e8ea26b7ecc41b3bf151525a2

**Comment by jhsu802701** [comment]
OK, I see that the branch is synced, but Gemfile.lock still shows the growl gem in place but not the libnotify file.  The changes your pull request makes in Gemfile.lock are at https://github.com/OpenTwinCities/adopt-a-tree/commit/991a9638abc37d7e8ea26b7ecc41b3bf151525a2#diff-e79a60dc6b85309ae70a6ea8261eaf95.

Please reverse those changes.  (The growl gem was added, and the libnotify gem was removed.)  Although the app won't pass in your Mac OS setup, you'll see it pass in Travis CI.)  The Gemfile specifies libnotify for the Linux environment (including the Travis CI environment) and growl for the Mac OS (Darwin) environment.

File: Gemfile.lock [hunk 1]
```diff
@@ -158,6 +159,7 @@ GEM
     git (1.5.0)
     globalid (0.4.2)
       activesupport (>= 4.2.0)
+    growl (1.0.3)
```
Comment (jhsu802701) [inline Gemfile.lock]: Delete

**Comment by jhsu802701** [comment]
I reverted all the other changes in Gemfile.lock.  Now it's passing the tests.

**Comment by jhsu802701** [comment]
I tried set up this branch on my local machine and tried out the app in my browser.  I can now see the terms of service.  Thanks, Corey!

**Comment by cgibsonmm** [comment]
Hey, thank you sorry I wasn’t more of a help been busy with life the last few weeks.

**Pull request merged**